components/dial/stories/dial.stories.js

```diff
@@ -1,11 +1,12 @@
-// Import the component markup template
 import { Template } from "./template";
 
 export default {
 	title: "Components/Dial",
 	description: "A dial is an input control used for selecting a value within a range, similar to a slider. It's often used in audio and video mixing and editing applications, where horizontal space is limited.",
 	component: "Dial",
 	argTypes: {
+		/* No theme styles for express available */
+		express: { table: { disable: true } },
```

Comment:
There is a different look for this one [on prod for Express](https://opensource.adobe.com/spectrum-css/preview/?path=/story/components-dial--default-express), I'm a bit confused on that

Reply:
Probably influenced by the global token values. My goal was adding express for components where we are adding local overrides since that's what we're maintaining.